### PR TITLE
apply summit tz to event feedback created col

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3040374&fileview=grid

### CODE REVIEW

File: src/actions/event-actions.js [hunk 1]
```diff
@@ -847,6 +847,7 @@ export const getEventFeedback =
         const {currentSummitState} = getState();
         const accessToken = await getAccessTokenSafely();
         const {currentSummit} = currentSummitState;
+        const summitTZ = currentSummit.time_zone.name;
```
Comment (smarcet) [inline src/actions/event-actions.js:850]: @romanetar check this https://github.com/fntechgit/summit-admin/commit/6a3dee53957e7447e1bb55bfdee8975c601512e2
we user time_zone_id
regards

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: src/actions/event-actions.js (hunk 1 above)
Comment (romanetar) [reply src/actions/event-actions.js]: done

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**